src/VirtualClient/VirtualClient.Actions/MLPerf/MLPerfExecutor.cs

```diff
@@ -220,8 +231,12 @@ await this.ThrowIfUnixDistroNotSupportedAsync(cancellationToken)
                 // add user in docker group and create scratch space
                 await this.ExecuteCommandAsync("usermod", $"-aG docker {this.Username}", this.NvidiaDirectory, cancellationToken);
                
-                // If GPUConfig is not included in the MLPerf code but is supported
-                this.ReplaceGPUConfigFilesToSupportAdditionalGPUs();
+                if (this.RequireCustomSystemSupport)
+                {
+                    // This enables A100_PCIe_80GBx4 system support that was not supported by github repo of MLPerf..
+                    this.ReplaceGPUConfigFilesToSupportAdditionalGPUs();
```

Comment:
If there is no harm in doing this, we can make it default

Reply:
Considering further GPUs like H100 or later, these files are getting updated in original repo, so it is better that user use files directly from repo. We have added these for A100 to support additional systems along with what they support.